src/ui/components/Library/Team/View/TestRuns/TestRunListItem.tsx

```diff
@@ -80,9 +81,6 @@ export function TestRunListItem({ testRun }: { testRun: TestRun }) {
           <RunStats testRun={testRun} />
         </div>
         <Attributes testRun={testRun} />
-        {testRun.title ? (
-          <div className="flex flex-row items-center justify-between text-xs">{testRun.title}</div>
-        ) : null}
```

Comment:
Background: The test run title is an optional field that the user could specify via environment variable in CI. We added it in response to cases in which the same test suite would be run multiple times (e.g. a matrix CI config) so the user could distinguish between multiple runs for the same PR and commit.

Reply:
I'd prefer we roll that into the `title` attribute rather than having multiple, optional titles (currently `commitTitle`, `mergeTitle`, `title`) – that's what my proposal currently pitches at least – any objection?